pkg/aws/client_test.go

```diff
@@ -587,4 +587,28 @@ var _ = Describe("Client", func() {
 			})
 		})
 	})
+
+	Context("AvailabilityZoneType", func() {
+
+		zoneName := "us-east-1a"
+
+		It("Fetches", func() {
+			input := &ec2.DescribeAvailabilityZonesInput{
+				ZoneNames: []*string{awsSdk.String(zoneName)},
+			}
+			output := &ec2.DescribeAvailabilityZonesOutput{
+				AvailabilityZones: []*ec2.AvailabilityZone{
+					{
+						ZoneType: awsSdk.String(LocalZone),
+					},
+				},
+			}
+
+			mockEC2API.EXPECT().DescribeAvailabilityZones(input).Return(output, nil)
+
+			zoneType, err := client.GetAvailabilityZoneType(zoneName)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(zoneType).To(Equal(zoneType))
```

Comment:
@marcolan018 I'd expect you to be comparing `zoneType` to some other variable and not itself, otherwise you're not verifying the value returned from the function is correct.

Reply:
@robpblake 
that's a typo. Thanks for pointing it out. Fixed it